cypress/e2e/tests/pages/extensions-compatibility-tests/kubewarden.spec.ts

```diff
@@ -0,0 +1,114 @@
+// import ExtensionsPagePo from '@/cypress/e2e/po/pages/extensions.po';
```

Comment:
probably worth dropping the file

Reply:
I'll drop the whole spec. I'll add it from another PR. This was still WIP

Reply:
https://github.com/rancher/dashboard/pull/11626/files